policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl

```diff
@@ -6,8 +6,13 @@
     strategy:
       fail-fast: false
       matrix:
-  {{- if and .Branchvals.Cgo (has "el7"  .Branchvals.Features) }}{{/* release-3.x version uses the same image for both el7 and non el7 */}}
+  {{- if and .Branchvals.Cgo (has "el7"  .Branchvals.Features) }}
+    {{- if eq .Branch "release-5-lts" }}
+        golang_cross: [ {{ .Branchvals.GoVersion }}-bullseye, {{ .Branchvals.GoVersion }}-el7 ]
```

Comment:
```suggestion
        golang_cross: [ {{ .Branchvals.GoVersion }}, 1.16-el7 ]
```

Reply:
So, the regular goversion(buildenv) will follow config, and that for el7 will have to be kept track of /updated from within the templates.
It makes the templates a bit unreadable, but I was trying this:
`golang_cross: [ {{ .Branchvals.GoVersion }}, {{ (.Branchvals.GoVersion | split "-")._0 }}-el7 ]`
it works for both `goversion` and `goversion-debversion` tags but the caveat is that it might be a bit unreadable.
I'm alright with either of the two options. What do you suggest?

Reply:
Approaches trying to compute the el7 version are solving a problem that we do not have. el7 can be built on _only one_ version of golang-cross.

Reply:
We'll have to still separately maintain two of them, because `release-4-lts` el7 packages are built using `1.15-el7` - https://github.com/TykTechnologies/tyk/blob/6831b76ee3953aa22968e92f37a8bf89fd4e4759/.github/workflows/release.yml#L45

Reply:
Oh, I forgot about that. But I think now that we have branch based conditionals, it can be handled there?
What I'd like is for the templates to be readable and also manage future release (lts) branches.